dgm_legacy/generate-finder-background.py

```diff
@@ -0,0 +1,81 @@
+from ij import IJ, ImagePlus
```

Comment:
Should this directory be called `dmg_legacy`? If so, I suggest rewriting the history to fix this typo.